tests/test_expval.py

```diff
@@ -15,53 +15,35 @@
 Unit tests for the :mod:`pennylane.plugin.DefaultGaussian` device.
 """
 # pylint: disable=protected-access,cell-var-from-loop
-import unittest
-import inspect
-import logging as log
-
-import pennylane as qml
-
 from pennylane import numpy as np
 from scipy.linalg import block_diag
 
-from defaults import pennylane as qml, BaseTest
+from defaults import pennylane as qml
```

Comment:
Once we have moved completely over to pytest, it should be fine to simply have `import pennylane as qml` here, since pytest has much better test discovery and import ability than unittest.